python/cuml/utils/numba_utils.py

```diff
@@ -183,3 +184,25 @@ def stride_from_order(shape, order, itemsize):
     if order == 'C':
         stride.reverse()
     return tuple(stride)
+
+
+# CuPy < 7.0 and numba >= 0.46 mitigation
```

Comment:
It would be nice to know from this comment what exactly is happening when these versions clash. Maybe add a tadd more detail here for other developers.

Reply:
its a cuda_array_interface conflict, will add more details in it